src/lib/addons/datadog-definition.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const {
+    FEATURE_CREATED,
+    FEATURE_UPDATED,
+    FEATURE_ARCHIVED,
+    FEATURE_REVIVED,
+    FEATURE_STALE_ON,
+    FEATURE_STALE_OFF,
+} = require('../types/events');
+
+module.exports = {
+    name: 'datadog',
+    displayName: 'Datadog',
+    description: 'Allows Unleash to post updates to Datadog.',
+    documentationUrl: 'https://docs.getunleash.io/docs/addons/datadog',
+    parameters: [
+        {
+            name: 'url',
+            displayName: 'Datadog Events URL',
+            description:
+                'Default url: https://api.datadoghq.com/api/v1/events. Needs to be changed if your not using the US1 site.',
+            type: 'url',
+            required: false,
+        },
+        {
+            name: 'apiKey',
+            displayName: 'DD API KEY',
+            placeholder: 'j96c23b0f12a6b3434a8d710110bd862',
+            description: 'Api key from datadog',
+            type: 'text',
+            required: true,
+            sensitive: true,
+        },
+    ],
+    events: [
+        FEATURE_CREATED,
+        FEATURE_UPDATED,
+        FEATURE_ARCHIVED,
+        FEATURE_REVIVED,
+        FEATURE_STALE_ON,
+        FEATURE_STALE_OFF,
+    ],
+    tagTypes: [
+        {
+            name: 'datadog',
+            description:
+                'Datadog tag used by the datadog-addon to specify tags to datadog event stream.',
+            icon: 'D',
+        },
+    ],
```

Comment:
The addon actully sends all tags to datadog for the feature toggle being changed  (which feels correct), do we then need to specify a specific tag-type for datadog?

Reply:
I have updated this.